astropy/units/tests/test_quantity_ufuncs.py

```diff
@@ -31,7 +31,7 @@ def test_testcase(tc):
     results = (results,) if not isinstance(results, tuple) else results
     for result, expected in zip(results, tc.q_out):
         assert result.unit == expected.unit
-        assert_allclose(result.value, expected.value, atol=1.0e-15)
+        assert_allclose(result.value, expected.value, atol=1.5e-8)
```

Comment:
`1.5e-8` seems very large. Was this meant to be `1.5e-15`? We could also just remove the `atol` altogether, since really we're just testing that things are more or less OK.

Reply:
I set the atol based on the reported absolute differences in the test log. Setting it to 1e-15 would fail the test. This is the actual log output. So what would you recommend? It only fails for one of the parametrized test case (`tc7`):

```
E           AssertionError: 
E           Not equal to tolerance rtol=1e-07, atol=1e-15
E           
E           Mismatched elements: 1 / 3 (33.3%)
E           Max absolute difference: 1.49011612e-08
E           Max relative difference: 2.82715972e-16
E            x: array([1.490116e-08, 7.853982e-01, 1.570796e+00])
E            y: array([0.      , 0.785398, 1.570796])
```

Example log from `main`: https://github.com/astropy/astropy/actions/runs/4906790496/jobs/8761477734

Reply:
Hmm, am actually surprised it differs from 0 that much, but probably best to just go with what you have...

Reply:
Should I open a follow-up issue for this?

Reply:
Maybe fine not to bother...